cldfbench_nominalperson_cldf.py

```diff
@@ -10,8 +11,8 @@ class Dataset(BaseDataset):
 
     def cldf_specs(self):  # A dataset must declare all CLDF sets it creates.
         from cldfbench import CLDFSpec
-        return CLDFSpec(dir=self.cldf_dir, module='StructureDataset') 
-#        return super().cldf_specs()
+        return CLDFSpec(dir=self.cldf_dir, module='Generic')  # As long as we don't add values.
+        return CLDFSpec(dir=self.cldf_dir, module='StructureDataset')
```

Comment:
A StructureDataset without values is invalid. Thus, for the time being, we stick to a Generic dataset.